packages/components/pages/f-registration/src/components/_tests/Registration.test.js

```diff
@@ -469,7 +469,7 @@ describe('Registration', () => {
 
             it('should show error message and emit failure event when the password field is populated with too short an input', async () => {
                 // Arrange
-                wrapper.find('[data-test-id="formfield-password-input"]').setValue('dog');
+                wrapper.find('[data-test-id="formfield-password-input"]').setValue('foobarbaz');
```

Comment:
I think a bigger refactor would be better in that we move all the validation logic to a separate class rather than leaving it in the template ... we did this in the 'f-account-info' component.

Reply:
Thanks Billy - I'll take a look at `f-account-info` and see how that was done. Will leave this PR as it currently is and see if we could spin up a small ticket to refactor 👍